### PR TITLE
Object Caching improvements

### PR DESCRIPTION
**Pull request by tillkruss**
1. Match wording of previous headlines
2. Improve plugin descriptions
3. Removed irrelevant links
4. Move Memcached plugin down, because it shows only 10 active installations
5. Added missing "Object Caching" section

### CODE REVIEW

**Comment by tillkruss** [comment]
Do I need to update the changelog when making changes?

File: performance/cache.md [hunk 1]
```diff
@@ -28,8 +36,6 @@ Adding an opcode cache like [Opcache](https://www.php.net/manual/en/book.opcache
 
 ## Further Reading {#further-reading}
 
-* [W3 Total Cache Plugin](http://dougal.gunters.org/blog/2009/08/26/w3-total-cache-plugin) (by Dougal Campbell)
-* [Holy Shmoly!: WP Super Cache](http://ocaoimh.ie/wp-super-cache/)
```
Comment (tillkruss) [inline performance/cache.md:32]: We're already mentioning plugins above.

File: performance/cache.md [hunk 2]
```diff
@@ -18,6 +18,14 @@ If your posts/pages have a lot of dynamic content configuring caching can be mor
...
+
+In addition to making data access much easier, cached data should always be replaceable and regenerable. If an application experiences database corruption (e.g., MySQL, Postgres, Couchbase), there will and should be severe consequences for this database (and let us hope that there is a good backup plan in place). In contrast with the main data store for the application, if a cache is corrupted, the application should continue to function as the cached data should regenerate itself. No data will be lost, although there will likely be some performance problems as the cache regenerates.
+
+The storage engine for an object cache can be a number of technologies. Popular object caching engines include Redis, Memcached, APC, and the file system. The caching engine used should be dictated by the needs of the application. Each has its advantages and disadvantages. At a bare minimum the engine used should make accessing the data more performant than regenerating the data.
```
Comment (tillkruss) [inline performance/cache.md:27]: Copied from: https://www.tollmanz.com/core-caching-concepts-in-wordpress

File: performance/optimization.md [hunk 3]
```diff
@@ -162,22 +162,20 @@ Look into HTTP Cache-Control (specifically `max-age`) and Expires headers, as we
...
 
-- [Memcached Object Cache](https://wordpress.org/extend/plugins/memcached/) – Provides a persistent backend for the WordPress object cache. A memcached server and the PECL memcached extension are required.
-- [Redis Object Cache](https://wordpress.org/plugins/redis-cache/) – Provides a consistent Redis object cache backend for WordPress that works with various Redis clients. A Redis Server is required.
+- [Redis Object Cache](https://wordpress.org/plugins/redis-cache/) – Provides a persistent Redis object cache backend for WordPress that works with various Redis clients. A Redis server is required.
+- [Memcached Object Cache](https://wordpress.org/extend/plugins/memcached/) – Provides a persistent Memcached object cache backend for WordPress. A memcached server and the [memcache](https://pecl.php.net/package/memcache) extension are required.
 
 ### Further Reading
 
-- [W3 Total Cache Plugin](http://dougal.gunters.org/blog/2009/08/26/w3-total-cache-plugin) (by Dougal Campbell)
-- [Holy Shmoly!: WP Super Cache](http://ocaoimh.ie/wp-super-cache/)
```
Comment (tillkruss) [inline performance/optimization.md:177]: Already mentioned above

File: performance/optimization.md [hunk 4]
```diff
@@ -162,22 +162,20 @@ Look into HTTP Cache-Control (specifically `max-age`) and Expires headers, as we
...
 
-- [Memcached Object Cache](https://wordpress.org/extend/plugins/memcached/) – Provides a persistent backend for the WordPress object cache. A memcached server and the PECL memcached extension are required.
-- [Redis Object Cache](https://wordpress.org/plugins/redis-cache/) – Provides a consistent Redis object cache backend for WordPress that works with various Redis clients. A Redis Server is required.
+- [Redis Object Cache](https://wordpress.org/plugins/redis-cache/) – Provides a persistent Redis object cache backend for WordPress that works with various Redis clients. A Redis server is required.
+- [Memcached Object Cache](https://wordpress.org/extend/plugins/memcached/) – Provides a persistent Memcached object cache backend for WordPress. A memcached server and the [memcache](https://pecl.php.net/package/memcache) extension are required.
```
Comment (javiercasares) [inline performance/optimization.md:172]: Proper URL:
https://wordpress.org/plugins/memcached/

**Pull request merged**